src/helpers/queries.js

```diff
@@ -134,3 +134,22 @@ export const buildTriggerQueryByName = name => ({
 export const buildTriggerQueryById = id => ({
   definition: Q(DOCTYPE_TRIGGERS).getById(id)
 })
+
+export const buildContactQueryUpdateGreaterThan = date => ({
+  definition: Q(DOCTYPE_CONTACTS)
+    .where({
+      trashed: {
+        $exists: false
+      },
+      $or: [
+        {
+          cozyMetadata: { $exists: false }
```

Comment:
Is this predicate really necessary? If not, the query could be way more efficient by simply indexing on `cozyMetadata.updatedAt`